content/de/docs/tutorials/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Tutorials
+main_menu: true
+weight: 60
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+Dieser Abschnitt der Kubernetes-Dokumentation enthält Tutorials.
+Ein Tutorial zeigt, wie Sie ein Ziel erreichen, das größer ist als eine einzelne [Aufgabe](/docs/tasks/). 
+Ein Tutorial besteht normalerweise aus mehreren Abschnitten, die jeweils eine Abfolge von Schritten haben.
+Bevor Sie die einzelnen Lernprogramme durchgehen, möchten Sie möglicherweise ein Lesezeichen zur Seite mit dem [Standardisierten Glossar](/docs/reference/glossary/) setzen um später Informationen nachzuschlagen.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Grundlagen
+
+* [Kubernetes Basics](/docs/tutorials/kubernetes-basics/) ist ein ausführliches interaktives Lernprogramm, das Ihnen hilft, das Kubernetes-System zu verstehen und einige grundlegende Kubernetes-Funktionen auszuprobieren.
+
+* [Scalable Microservices mit Kubernetes (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615) (Englisch)
+
+* [Einführung in Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#) (Englisch)
+
+* [Hello Minikube](/docs/tutorials/hello-minikube/)
+
+## Konfiguration
+
+* [Redis mit einer ConfigMap konfigurieren](/docs/tutorials/configuration/configure-redis-using-configmap/)
+
+## Stateless Anwendungen
+
+* [Exposing an External IP Address to Access an Application in a Cluster](/docs/tutorials/stateless-application/expose-external-ip-address/)
```

Comment:
Is it possible to translate the link-text for this link and the links below as well (e.g. `Exposing an External IP Address to Access an Application in a Cluster`) ?